### PR TITLE
docs: Add Chinese README file

### PR DESCRIPTION
**Pull request by lhDream**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 ![promo](https://raw.githubusercontent.com/AlmasB/git-server/master/storage/images/fxgl_promo.jpg)
 
+## 语言
```
Comment (AlmasB) [inline README.md]: 语言 should be "Language" since this file is the English version

**Review by AlmasB — COMMENTED** [review]
Thanks a lot @lhDream , this is a great idea!

@chengenzhao as you kindly offered to help with #1172, can you help me review `README_CN` please.

File: README_CN.md [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+- [Adam Bocco](https://github.com/adambocco)
+
+测试人员：
+
+- [Carl Dea](https://github.com/carldea)
+- [Frank Delporte](https://github.com/FDelporte)
+
+### 贡献与支持
+
+如果您想从源代码构建 FXGL 或想贡献，请参阅[贡献指南](https://github.com/lhDream/FXGL/blob/dev/CONTRIBUTING.md)（包括非代码）。FXGL 是完全模块化的，因此新的贡献者不需要了解整个代码库，只需要了解做出贡献的模块即可。贡献将根据[行为准则](https://github.com/lhDream/FXGL/blob/dev/CODE_OF_CONDUCT.md)进行审查。
```
Comment (AlmasB) [inline README_CN.md]: I think these links (contributing and code of conduct) need to be rerouted to main repo.

File: README.md (hunk 1 above)
Comment (chengenzhao) [reply README.md]:
```suggestion
## Languages
```

File: README_CN.md [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+
+JavaFX 游戏开发框架
+
+[![Maven Central](https://img.shields.io/maven-central/v/com.github.almasb/fxgl.svg)]()[![Build Status](https://github.com/AlmasB/FXGL/workflows/Java%20CI%20with%20Maven/badge.svg)](https://github.com/AlmasB/FXGL/actions)[![codecov](https://codecov.io/gh/AlmasB/FXGL/branch/dev/graph/badge.svg)](https://codecov.io/gh/AlmasB/FXGL)[![sponsor](https://img.shields.io/badge/sponsor-%241-brightgreen)](https://github.com/sponsors/AlmasB)
+
+## 为什么选择FXGL ?
+
+- 无需安装或设置
+- “开箱即用”：Java 8-17、Win/Mac/Linux/Android 8+/iOS 11.0+/Web
+- 简单干净的API，比其他引擎更高级别
```
Comment (chengenzhao) [inline README_CN.md]:
```suggestion
- 简单干净的API，相比起其他引擎API级别更高
```

File: README_CN.md [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+- JavaFX 的超集：无需学习新的 UI API
+- 真实世界的游戏开发技术：实体组件、插值动画、粒子[等等](https://github.com/AlmasB/FXGL/wiki/Core-Features)
+- 游戏很容易打包成一个可执行的 .jar 或原生镜像
+
+### 适合：
+
+- 任何 2D 游戏（横向卷轴 /platformer/ 街机 / RPG游戏）
+- 任何具有复杂 UI 控件/动画的业务应用程序
+- 实验 3D
```
Comment (chengenzhao) [inline README_CN.md]:
```suggestion
- 实验性 3D 特性
```

File: README_CN.md [hunk 5]
```diff
@@ -0,0 +1,184 @@
...
+    }
+}
+```
+
+## 入门
+
+如需快速深入了解代码，请参阅独立的[基本示例](https://github.com/AlmasB/FXGL/tree/dev/fxgl-samples/src/main/java/basics)。否则，请参阅：
+
+- [展示预告片](https://youtu.be/fuDQg7W0v4g)
+- [将 FXGL 与 IntelliJ 一起使用](https://youtu.be/LhmlFC6KE2Q)
```
Comment (chengenzhao) [inline README_CN.md]:
```suggestion
- [与 FXGL 与 IntelliJ 一起使用](https://youtu.be/LhmlFC6KE2Q)
```

File: README_CN.md [hunk 6]
```diff
@@ -0,0 +1,184 @@
...
+        launch(args);
+    }
+}
+```
+
+## 入门
+
+如需快速深入了解代码，请参阅独立的[基本示例](https://github.com/AlmasB/FXGL/tree/dev/fxgl-samples/src/main/java/basics)。否则，请参阅：
+
+- [展示预告片](https://youtu.be/fuDQg7W0v4g)
```
Comment (chengenzhao) [inline README_CN.md]:
```suggestion
- [预告片展示](https://youtu.be/fuDQg7W0v4g)
```

File: README_CN.md [hunk 7]
```diff
@@ -0,0 +1,184 @@
...
+- [将 FXGL 与 IntelliJ 一起使用](https://youtu.be/LhmlFC6KE2Q)
+- [维基和书面教程](https://github.com/AlmasB/FXGL/wiki)
+- [YouTube 教程](https://www.youtube.com/playlist?list=PL4h6ypqTi3RTiTuAQFKE6xwflnPKyFuPp)
+- [示例代码演示](https://github.com/lhDream/FXGL/blob/dev/fxgl-samples)
+- [FXGL 游戏](https://github.com/AlmasB/FXGLGames)（附源码）
+- 在 `itch.io `上发布了[演示](https://fxgl.itch.io/)
```
Comment (chengenzhao) [inline README_CN.md]:
```suggestion
- [演示](https://fxgl.itch.io/)在 `itch.io `上发布
```

File: README_CN.md [hunk 8]
```diff
@@ -0,0 +1,184 @@
...
+
+```
+<dependency>
+    <groupId>com.github.almasb</groupId>
+    <artifactId>fxgl</artifactId>
+    <version>17.1</version>
+</dependency>
+```
+
+注意：`0.5.4`用于 Java 8-10 和`11.17`Java 11-16。
```
Comment (chengenzhao) [inline README_CN.md]:
```suggestion
注意：`0.5.4` 及 `11.17` 分别适用于 Java 8-10  及 Java 11-16。
```

File: README_CN.md [hunk 9]
```diff
@@ -0,0 +1,184 @@
...
+- [将 FXGL 与 IntelliJ 一起使用](https://youtu.be/LhmlFC6KE2Q)
+- [维基和书面教程](https://github.com/AlmasB/FXGL/wiki)
+- [YouTube 教程](https://www.youtube.com/playlist?list=PL4h6ypqTi3RTiTuAQFKE6xwflnPKyFuPp)
+- [示例代码演示](https://github.com/lhDream/FXGL/blob/dev/fxgl-samples)
+- [FXGL 游戏](https://github.com/AlmasB/FXGLGames)（附源码）
+- 在 `itch.io `上发布了[演示](https://fxgl.itch.io/)
+
+### Maven
+
+- [Maven](https://github.com/AlmasB/FXGL-MavenGradle)模板项目，如果你使用 Java 和/或 Kotlin (Java 17+)
```
Comment (chengenzhao) [inline README_CN.md]:
```suggestion
- [Maven](https://github.com/AlmasB/FXGL-MavenGradle)项目模板，编程语言为 Java 和/或 Kotlin (Java 17+)
```

File: README_CN.md [hunk 10]
```diff
@@ -0,0 +1,184 @@
...
+    <artifactId>fxgl</artifactId>
+    <version>17.1</version>
+</dependency>
+```
+
+注意：`0.5.4`用于 Java 8-10 和`11.17`Java 11-16。
+
+### Gradle
+
+- 如果您使用 Java 和/或 Kotlin (Java 17+)，则为[Gradle模板项目](https://github.com/AlmasB/FXGL-MavenGradle)
```
Comment (chengenzhao) [inline README_CN.md]:
```suggestion
- [Gradle项目模板](https://github.com/AlmasB/FXGL-MavenGradle)，编程语言为 Java 和/或 Kotlin (Java 17+)
```

File: README_CN.md [hunk 11]
```diff
@@ -0,0 +1,184 @@
...
+repositories {
+    jcenter()
+}
+
+dependencies {
+    compile 'com.github.almasb:fxgl:17.1'
+}
+```
+
+注意：`0.5.4`用于 Java 8-10 和`11.17`Java 11-16。
```
Comment (chengenzhao) [inline README_CN.md]:
```suggestion
注意：`0.5.4` 及 `11.17` 分别适用于 Java 8-10  及 Java 11-16。
```

File: README_CN.md [hunk 12]
```diff
@@ -0,0 +1,184 @@
...
+
+如果您希望开发模块化应用程序，这里有一个完整的示例`module-info.java`：
+
+```
+open module app.name {
+    requires com.almasb.fxgl.all;
+}
+```
+
+### Uber jar
```
Comment (chengenzhao) [inline README_CN.md]:
```suggestion
### 独立完整包含所有依赖的 jar 文件
```

File: README_CN.md [hunk 13]
```diff
@@ -0,0 +1,184 @@
...
+
+- 英国布莱顿大学
+- 英国诺丁汉大学
+- 美国佐治亚理工学院
+- 美国kidOYO
+- 美国沃尔顿高中
+- 美国科南特高中
+- 丹麦新西兰商业技术学院
+- 巴西南里奥格兰德联邦教育、科学和技术学院
+- FHNW 工程/计算机科学学院，瑞士
```
Comment (chengenzhao) [inline README_CN.md]:
```suggestion
- 瑞士西北高等专业学院 工程/计算机科学学院
```

File: README_CN.md [hunk 14]
```diff
@@ -0,0 +1,184 @@
...
+- @sabit86
+- @hendrikebbers
+- @ImperaEtConquer
+- @thejeed
+
+公司：
+
+- @karakun
+
+### 接触
```
Comment (chengenzhao) [inline README_CN.md]:
```suggestion
### 与我们取得联系
```

**Review by chengenzhao — CHANGES_REQUESTED** [review]
根据一些语言习惯和上下文环境，做了相应的修改建议
Based on language custom and context, provided some suggestions

**Comment by AlmasB** [comment]
Thank you both.

@chengenzhao let me know if ok to merge.

**Comment by chengenzhao** [comment]
@lhDream 还有两个修改建议，是遗漏了还是保留原先的翻译？

**Comment by lhDream** [comment]
@chengenzhao 两个？那应该是漏了一个,等下我瞅一眼。在考虑用原来的还是换成这个 `在IntelliJ中使用FXGL`

**Comment by chengenzhao** [comment]
> 

我觉得：在IntelliJ中使用FXGL
这个翻译比较好，改好就行了，我先approve一下

**Review by chengenzhao — APPROVED** [review]

**Comment by lhDream** [comment]
OK

**Comment by chengenzhao** [comment]
@AlmasB the doc is good to go

**Comment by AlmasB** [comment]
Great, many thanks!

**Pull request merged**